### PR TITLE
add regedit support and associated yaml

### PR DESCRIPTION
**Pull request by SaltyCharles**
similar to secedit with a few modifications specific to regedit
responses.  not complete coverage (yet)

### CODE REVIEW

File: hubblestack_nova/regedit.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        ret['Success'] = success
+        ret['Failure'] = failure
+
+    if not ret['Controlled']:
+        ret.pop('Controlled')
+
+    return ret
+
+
+def _findOptionValueInReg(reg_hive, reg_key, reg_value):
```
Comment (basepi) [inline hubblestack_nova/regedit.py]: `_find_option_value_in_reg` is more pythonic

**Comment by basepi** [comment]
I left a comment inline. (which I see you've already handled)

Could you also rename `regedit.py` to `win_regedit.py`? I think we're going to distinguish windows-only modules this way. (I plan to fix secedit shortly)

Are these audits from a governing body or just sample audits to show that you can audit reg values and whatnot? Maybe we should rename your yaml file to something like `MISC-Win-2012-R2.yaml`.

**Comment by SaltyCharles** [comment]
No problem at all, I'll make those changes real quick.  No, this is not from a governing body.  I'll modify the file and add one for CIS shortly.

**Comment by basepi** [comment]
Perfect!

**Pull request merged**